resources/views/frontend/catalog-new/index-new.blade.php

```diff
@@ -10,7 +10,7 @@
                         {{-- Desktop filter --}}
                         <filter-new-popover.group-controller>
                             <div
-                                class="tw-hidden tw-gap-x-3 tw-overflow-x-auto md:tw-flex md:tw-flex-wrap md:tw-overflow-visible">
+                                class="tw-z-50 tw-hidden tw-gap-x-3 tw-overflow-x-auto md:tw-flex md:tw-flex-wrap md:tw-overflow-visible">
```

Comment:
Would this work? Why does it need such a high number?
```suggestion
                                class="tw-z-20 tw-hidden tw-gap-x-3 tw-overflow-x-auto md:tw-flex md:tw-flex-wrap md:tw-overflow-visible">
```

Reply:
sorry my bad. it was left out there by accident